mesonbuild/msubprojects.py

```diff
@@ -74,14 +105,14 @@ def __init__(self, logger: Logger, r: Resolver, wrap: PackageDefinition, repo_di
         self.wrap = self.wrap_resolver.wrap = wrap
```

Comment:
What should I do about the line above?

Seems to be an abomination where self.wrap_resolver is some copy of `Resolver`, but has an extra field.